src/config/Config.ts

```diff
@@ -14,29 +14,31 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import { DEFAULT, IConfigOptions } from "./ConfigOptions";
+import { IConfigOptions } from "./ConfigOptions";
 
 export class Config {
-  static instance: Config;
-  static init(): Promise<void> {
-    if (Config?.instance?.initPromise) {
-      return Config.instance.initPromise;
+  private static _instance: Config;
```

Comment:
```suggestion
  private static internalInstance: Config;
```

(Personal preference)